src/main/java/Main.java

```diff
@@ -0,0 +1,21 @@
+import core.basesyntax.dataprocess.DataProcessor;
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.DataReaderService;
+import core.basesyntax.service.DataWriterService;
+import core.basesyntax.service.impl.DataReaderServiceImpl;
+import core.basesyntax.service.impl.DataWriterServiceImpl;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
see main() method structure - all the steps should be present
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/eaaf5ff3-fe0b-47e5-9731-c39b720c4666)